base/generator.jl

```diff
@@ -20,6 +20,12 @@ function next(g::Generator, s)
     g.f(v), s2
 end
 
+"""
+    generate(f, c...) -> iterator
+
+Apply `f` to each element of `c`.
+For multiple collection arguments, apply f elementwise.
+"""
 generate(f, c...) = Generator(f, c...)
```

Comment:
I see your point. Clearly, we don't know what policy to follow as regards the naming of iterators and functions that create them...

Anyway, the docs should explicitly say "Return an iterator applying `f` to each element of `c`.".